Storage/src/Connection/ServiceDefinition/storage-v1.json

```
@@ -745,6 +745,10 @@
                         "$ref": "Bucket"
                     }
                 },
+                "unreachable": {
+                    "type": "array",
+                    "description": "A list of bucket names (as strings) from locations that could not be reached."
+                },
```

Based on the discovery [docs](https://storage.googleapis.com/$discovery/rest?version=v1), shouldn't this be something like this?
```suggestion
                "unreachable": {
                    "type": "array",
                    "description": "The list of bucket resource names that could not be reached during the listing operation.",
                    "items": {
                        "type": "string"
                     }
                },
```

---

Hi @nidhiii-27 yes, I agree .
Did the required changes.